unit_tests/sources/declarative/datetime/test_min_max_datetime.py

```diff
@@ -22,13 +22,13 @@
         (
             "test_time_is_greater_than_min",
             "{{ config['older'] }}",
-            "{{ stream_state['newer'] }}",
+            "{{ stream_slice['start_date'] }}",
```

Comment:
We've been recommending people use the dedicated `stream_interval` or `stream_partition` variables instead of `stream_slice`. Lets do that in tests too.

But just to note, we still want to pass in `**{"stream_slice": stream_slice}` in the kwargs since that is how our code passed the context variables in, but we want to test the interpolation block aliases to `stream_interval` since thats how most users should use it.